tests/integration-tests/tests/update/test_update.py

```diff
@@ -1482,3 +1482,67 @@ def _wait_until_instances_are_stopped(cluster):
         logging.info("Still found %2d compute instances in the cluster. Waiting... ", n_compute_instances)
 
     return n_compute_instances <= 1
+
+
+@pytest.mark.usefixtures("instance")
+def test_login_nodes_count_update(os, pcluster_config_reader, clusters_factory, test_datadir):
+    """Test cluster Updates (add, remove LN pools)."""
+
+    # Start a cluster without login nodes
+    initial_config = pcluster_config_reader(config_file="pcluster_create_without_login_nodes.config.yaml")
+    cluster = clusters_factory(initial_config)
+
+    # Describe cluster, verify the response is without login node section
+    cluster_info = cluster.describe_cluster()
+    assert_that(cluster_info).is_not_none()
+    assert_that(cluster_info).does_not_contain("loginNodes")
+
+    # Update the cluster adding 3 head node
```

Comment:
typo: adding 3 'login' nodes

Reply:
Thank you for reminding. Have corrected it.